scripts/xnat/check_new_sessions

```diff
@@ -280,7 +287,7 @@ frame_check_ge = { 'ncanda-t1spgr-v1' : '146',
                    'ncanda-grefieldmap-v1' : '384',
                    'ncanda-rsfmri-v1' : '8768' }
 
-def frame_check(eid, manufacturer):
+def frame_check(eid, manufacturer,label):
```

Comment:
@abonil91 include a space after the comma 

`def frame_check(eid, manufacturer, label):`